packages/http-client-csharp/CONTRIBUTING.md

```diff
@@ -21,39 +21,43 @@ Before you begin, ensure you have the following installed:
 - [Node.js](https://nodejs.org/) (version 20 or higher)
 - [.NET 9.0 SDK](https://dotnet.microsoft.com/download/dotnet/9.0) (required - see global.json for exact version)
 - [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [pnpm](https://pnpm.io/installation)
```

Comment:
We shouldn't need pnpm for our package. At least I didn't think we did. were you seeing issues?

Reply:
There's a note later on the guide that prompt the user to build the entire workspace using `pnpm`. Is that needed? If not, I can remove all mentions of `pnpm`.